core/pom.xml

```diff
@@ -74,6 +74,16 @@ the License.
         <artifactId>aws-java-sdk</artifactId>
         <version>1.7.4</version>
       </dependency>
+      <dependency>
+        <groupId>org.apache.avro</groupId>
+        <artifactId>avro-ipc</artifactId>
```

Comment:
Can you please make the ```avro.version``` a defined property within the ```<properties></properties>``` element. Then it can be used consistently.